lib/robot.js

```diff
@@ -25,6 +25,25 @@ class Robot {
     });
   }
 
+  addEventToListeners(event, callback) {
```

Comment:
> If its a class method, then it'll be accessible…

I was thinking the same thing, @JasonEtco. Even if we don't doc it, someone will see it's there and start using it.

A few ideas for avoiding adding a new method:

1. Move everything back into `events` and call itself recursively if the event name is an array. For example
    ```js
    on(event, callback) {
      if (typeof event === 'array') {
        event.forEach(event, e => this.on(e, callback));
        return;
      }

      const [name, action] = event.split('.');
      // …the rest of the code…
    ```
1. If you wanted to extract some code to keep the function from getting ginormous, the function that gets passed to `this.events.on` could be extracted to a method that gets exposed publicly (I was doing something hacky a few weeks ago and wished this bit of code was accessible). I'd need to think a little more about what to call this, but the method handles creating the github client, creating a context, and capturing any errors raised.
    ```js
    return this.events.on(name, this.handler(callback));
    ```